pkg/sdk/config.go

```diff
@@ -30,7 +42,11 @@ func ProfileConfig(profile string) (*gosnowflake.Config, error) {
 	var config *gosnowflake.Config
 	if cfg, ok := configs[profile]; ok {
 		log.Printf("[DEBUG] loading config for profile: \"%s\"", profile)
-		config = cfg
+		driverCfg, err := cfg.DriverConfig()
```

Comment:
I believe this fallback is not documented

Reply:
It's not really a fallback - we just use different types here. IMO, this is too low-level to be documented for the useres.